CRM/Mailing/Event/BAO/MailingEventForward.php

```diff
@@ -233,7 +233,7 @@ protected static function _civicrm_api3_deprecated_add_formatted_param(&$values,
 
     // get the formatted location blocks into params - w/ 3.0 format, CRM-4605
     if (!empty($values['location_type_id'])) {
-      static $fields = NULL;
+      $fields = NULL;
       if ($fields == NULL) {
         $fields = [];
       }
```

Comment:
Ditto for previous comment. In order to preserve the meaning I think all 4 lines should be removed.

Reply:
Updated as per your suggestion and recommitted the file.